src/rules/missing-playwright-await.ts

```diff
@@ -139,6 +206,21 @@ export default createRule({
 
     return {
       CallExpression(node) {
+        // Checking validity of calls to methods on the page object
+        if (isPage(node) && node.callee.type === 'MemberExpression') {
+          const method = getStringValue(node.callee.property)
+          const isValid = checkValidity(node)
+
+          if (!isValid && pageMethods.has(method)) {
+            context.report({
+              data: { method },
+              fix: (fixer) => fixer.insertTextBefore(node, 'await '),
+              messageId: 'page',
+              node: node.callee,
+            })
+          }
```

Comment:
Go ahead and return at the end of this if statement, if it is a page method, no need to run `parseFnCall` since it won't be a expect or test function